content/scalers/huawei-cloudeye.md

```diff
@@ -0,0 +1,120 @@
++++
+fragment = "content"
+weight = 100
+title = "Huawei Cloudeye"
+background = "light"
++++
+
+Scale applications based on a Huawei Cloudeye.
+
+**Availability:** v1.0+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `huawei-cloudeye` trigger that scales based on a Huawei Cloudeye.
+
+```yaml
+triggers:
+  - type: huawei-cloudeye
+    metadata:
+      namespace: SYS.ELB
+      dimensionName: lbaas_instance_id
+      dimensionValue: 5e052238-0346-xxb0-86ea-92d9f33e29d2
+      metricName: mb_l7_qps
+      targetMetricValue: "100"
+      minMetricValue: "1"
+```
+
+### Authentication Parameters
+
+You can use `TriggerAuthentication` CRD to configure the authenticate by providing a set of IAM credentials.
+
+**Credential based authentication:**
```

Comment:
That's interesting - Do you need all of these to authenticate or are there different combinations at play here?

Reply:
Thanks @tomkerkhove . At the moment, it is indeed like this for huaweicloud.

Reply:
That's OK, just to confirm - I'd need to specify all parameters, it's not that we need to provide a, b and c of d, e and f.

Feel free to resolve if it's all or nothing.